config/stargate.json

```diff
@@ -148,6 +151,10 @@
     {
       "chainId": 1101,
       "lzChainId": 158
+    },
+    {
+      "chainId": 534352,
+      "lzChainId": 214
     }
```

Comment:
@ezynda3 did you add the new id to the mappings on all all other chains supported by stargate?

Reply:
Yes